templates/kapeta/block-type-cli/prepare-hosts.ts.hbs

```diff
@@ -22,27 +21,25 @@ const RESOURCES = [
 (async function () {
     const config = await Config.init(BASE_PATH);
 
-    // Ensure the config directory exists
-    await mkdir(CONFIG_DIR, { recursive: true });
-
-    const hosts: { [key: string]: string } = {};
+    let envContents = '';
```

Comment:
Maybe make it start w/ a comment about being generated, so you're not caught off guard when it's overwritten

Reply:
e84f4f8e2a54d49c193ed8f02a8e356c6ead1152